doc/cheatsheet.md

```diff
@@ -0,0 +1,262 @@
+# djot cheatsheet
+
+## Basics
+
+| Markup | Result |
+| --- | --- |
+| `_italic_` or `{_italic_}` | *italic* |
+| `*bold*` or `{*bold*}` | **bold** |
+| `` `verbatim/code` `` | `verbatim/code` |
+| `H~2~O` | H<sub>2</sub>O |
+| `20^th^` | 20<sup>th</sup> |
+| `{=highlighted=}` | <mark>highlighted</mark> |
+| `{+insert+}` | <ins>insert</ins> |
+| `{-delete-}` | <del>delete</del> |
+
+Use triple-backticks for fenced verbatim blocks. If it's a code
+block, you may append the language name after the opening backticks
+to get syntax highlighting:
+
+~~~~~~
+```mylang
+func say-hello(nm) {
+    print("hello ${nm}!");
+}
+```
+~~~~~~
+
```

Comment:
I suggest creating a separate section heading for code blocks.

Reply:
:+1: